File: backend/src/main/java/ca/metricalsky/winston/entity/view/VideoStatisticsView.java
package ca.metricalsky.winston.entity.view;

import java.time.OffsetDateTime;

public interface VideoStatisticsView {

    String getChannelId();

    String getVideoId();

    Long getCommentCount();

    Long getReplyCount();

    OffsetDateTime getLastCommentedAt();
}

File: database/src/main/kotlin/ca/metricalsky/winston/database/view/VideoStatisticsView.kt
package ca.metricalsky.winston.database.view

import java.time.OffsetDateTime

interface VideoStatisticsView {

    fun getChannelId(): String

    fun getVideoId(): String

    fun getCommentCount(): Long

    fun getReplyCount(): Long

    fun getLastCommentedAt(): OffsetDateTime
}
